src/topic/TopicMessageSubmitTransaction.js

```diff
@@ -163,9 +163,19 @@ export default class TopicMessageSubmitTransaction extends Transaction {
      */
     setMessage(message) {
         this._requireNotFrozen();
-        this._message =
-            typeof message === "string" ? utf8.encode(message) : message;
-
+        if (message == undefined) {
```

Comment:
```suggestion
        if (message == null) {
```

Reply:
Since you're creating a
```javascript
if (value == null) {
    throw new Error("value cannot be null");
}
```
Maybe it'd be a good idea to move this into a separate function inside `./src/util.js` or some other file which does this exact check? Something like:
```javascript
/**
 * @template {*} T
 * @param {T | null | undefined} value
 * @returns {NonNullable<T>}
 */
function requireNonNull(value = undefined) {
    if (value == null) {
        throw new Error("value cannot be null");
    }

    return /** @type {NonNullable<T>} */ (value);
}
```